### PR TITLE
out_stackdriver: Document metadata_server_url config option

### PR DESCRIPTION
**Pull request by heartrobotninja**
for https://github.com/fluent/fluent-bit/pull/2944

Signed-off-by: Joey DeStefanis <jdestefanis@google.com>

### CODE REVIEW

**Comment by heartrobotninja** [comment]
@edsiper Here is the documentation for the stackdriver metadata merge.

File: pipeline/outputs/stackdriver.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Before to get started with the plugin configuration, make sure to obtain the pro
 | google\_service\_credentials | Absolute path to a Google Cloud credentials JSON file | Value of environment variable _$GOOGLE\_SERVICE\_CREDENTIALS_ |
 | service\_account\_email | Account email associated to the service. Only available if **no credentials file** has been provided. | Value of environment variable _$SERVICE\_ACCOUNT\_EMAIL_ |
 | service\_account\_secret | Private key content associated with the service account. Only available if **no credentials file** has been provided. | Value of environment variable _$SERVICE\_ACCOUNT\_SECRET_ |
+| metadata\_server | Prefix for a metadata server. Can also set environment variable _$METADATA\_SERVER_  | http://metadata.google.internal |
```
Comment (igorpeshansky) [inline pipeline/outputs/stackdriver.md:18]: When the description is multiple sentences, there's a trailing period.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
A minor nit.

File: pipeline/outputs/stackdriver.md (hunk 1 above)
Comment (heartrobotninja) [reply pipeline/outputs/stackdriver.md:18]: Fixed.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**